x/projects/keeper/migrations.go

```diff
@@ -25,3 +27,71 @@ func (m Migrator) Migrate2to3(ctx sdk.Context) error {
 	}
 	return nil
 }
+
+// Migrate3to4 implements protobuf migration from v3 to v4:
+// Now the project protobuf has admin and subscription policies
+func (m Migrator) Migrate3to4(ctx sdk.Context) error {
+	projectIndices := m.keeper.projectsFS.GetAllEntryIndices(ctx)
+	for _, projectIndex := range projectIndices {
+		blocks := m.keeper.projectsFS.GetAllEntryVersions(ctx, projectIndex, true)
+		for _, block := range blocks {
+			var oldProjectStruct v3.Project
+			if found := m.keeper.projectsFS.FindEntry(ctx, projectIndex, block, &oldProjectStruct); !found {
```

Comment:
Use `ReadEntry()` when we know the exact block version.

Reply:
same